credsweeper/scanner/scan_type/scan_type.py

```diff
@@ -83,7 +84,7 @@ def get_line_data(cls, config: Config, line: str, line_num: int, file_path: str,
         if not cls.is_valid_line(line, pattern, line_num, file_path):
             return None
         logger.debug("Valid line for pattern: %s in file: %s:%d in line: %s", pattern, file_path, line_num, line)
-        line_data = LineData(config, line, line_num, file_path, pattern)
+        line_data = LineData(config=config, line=line, line_num=line_num, path=file_path, info=info, pattern=pattern)
```

Comment:
Is there any difference between `config=config` and just type `config`?

It seems there are two styles in our code..... I think it would be good to merge to one style and I prefer just type `config`..

How do you think?

Reply:
Yes, there is just ``config`` possibly. 
Sometimes there are possibly mistakes when arguments have the same type, and might be swapped :(
I changed it in https://github.com/Samsung/CredSweeper/pull/232/commits/1bd8eb82819d20d3da6eabb63bd8b47f1313aa6f
- the same lines are touched.